### ISSUE TITLE
Controls are messed up

### ISSUE THREAD

**Issue by ProphetOfDuality** [CemuUser8/mouse2joystick_custom_CEMU#10]
I'm using this on Breath of the Wild and for some reason, the controls are all mixed up. Moving my mouse up causes my character to move left, holding 'A' makes him move back, stuff like that. I'm not sure if this is an issue with cemu or not but if anyone has any advice I'd really appreciate it.

**Comment by CemuUser8** [comment]
I know this was a very long time ago, but I am just now trying to get back into managing this project a little bit.

Do you know if you were using the PRO controller setup, and does switching to GamePad profile work better?

**Comment by ProphetOfDuality** [comment]
I actually tried downloading it again and everything works fine now. I must've done something wrong during the install or had another program that was messing with it. 

To answer your question, I had tried using all the different controller profiles and none of them worked. I think what was happening was that it was mapping all the buttons to the wrong places. Probably because of something on my part.
